system/mount.py

```diff
@@ -358,13 +358,17 @@ def main():
                     res,msg = mount(module, **args)
             elif 'bind' in args.get('opts', []):
                 changed = True
-                cmd = 'mount -l'
+                cmd = 'findmnt -rn %s' % args['name']
```

Comment:
To check if executable exists in the path, you can use this code:

```
import distutils.spawn
bin_path = distutils.spawn.find_executable('mount')
if bin_path is not None:
  ...
```

Works for Python 2.4+.